README.md

```diff
@@ -416,6 +416,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [helplightning.com](https://www.helplightning.com/) — Help over video with augmented reality. Free without analytics, encryption, support
   * [ideascale.com](https://ideascale.com/) — Allow clients to submit ideas and vote, free for 25 members in 1 community
   * [Igloo](https://www.igloosoftware.com/) — Internal portal for sharing documents, blogs, calendars, etc. Free for up to 10 users.
+  * [shareweb.app](https://shareweb.app) - A better way to report bugs for your website. It allows you to capture annotated screenshots, record videos, and replay user's session with the technical details engineers need to identify issues. Free for up to 10 issues per month.
```

Comment:
10 issues a month is not useful for any level of testing, sorry think I'll pass on this